DatabaseLink.java

```diff
@@ -0,0 +1,59 @@
+package com.company.dbconnection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class DatabaseLink extends Thread {

```

Comment:
This is completely wrong, the idea of the PoC is not for the Db operation to be a thread, it is a simple insert operation, you are not going to gain anything by making it a different thread. The thread should be used to search the file, thus if the file is very large, your application will not hang while the search is happening.

Reply:
I have changed the function of the thread to searching the word from linking to the DB